tools/rnc-validate-format.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import os
+import subprocess
+import argparse
+from shutil import which
+from glob import glob
+
+# to validate and pretty print the rnc schema files before commit:
+# tools/rnc-validate-format.py
+
+parser = argparse.ArgumentParser(description="process csl schema files")
+parser.add_argument("-c", action="store_true", help="commit the file(s)")
```

Comment:
i think long args help with readability, especially for stuff that is not going to be frequently typed but read a lot:

```suggestion
parser.add_argument("--commit", action="store_true", help="commit the file(s)")
```